.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: bash
+sudo: required
+service:
+- docker
+before_install:
+- docker build -t todo-app-test:latest .
```

Comment:
Is there any particular reason you've put this into the `before_install` stage? No particular harm, but in terms of intent I think this is a core part of your CI pipeline, so I'd simply include it as an initial step under `script`